kafka-connect-s3/pom.xml

```diff
@@ -67,7 +60,7 @@
         </dependency>
         <dependency>
             <groupId>io.findify</groupId>
-            <artifactId>s3mock_2.11</artifactId>
+            <artifactId>s3mock_2.12</artifactId>
```

Comment:
AK 2.1 uses Scala 2.11, whereas AK trunk (which our `master` currently corresponds to) uses Scala 2.12. So this should not be backported further back than `master`.

Reply:
using `${kafka.scala.version}` should make this portable across branches